.github/workflows/tests.yml

```diff
@@ -96,10 +96,14 @@ jobs:
       with:
         name: laravel-logs
         path: storage\logs\*.log
-    - name: Slack Notification
-      uses: rtCamp/action-slack-notify@v2
-      env:
-        SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+     - name: Send Slack notification
+       uses: 8398a7/action-slack@v2
```

Comment:
@gggrant Any reason to use v2 instead of [v3 (latest)](https://github.com/8398a7/action-slack) for the Slack notifications?

Reply:
Nope! I'll update when doing the browser tests.